test/dragAndDrop.test.js

```diff
@@ -0,0 +1,34 @@
+import dragAndDrop from './__mocks__/dragAndDropMock';
+import './__mocks__/localStorageMock';
+
+describe('dragAndDrop', () => {
+  it('should throw an error if no task is given', () => {
+    expect(() => {
+      dragAndDrop.editTask();
+    }).toThrow();
+  });
+
```

Comment:
- Great job on the edge cases 👍🏾 Kindly add test to check if a user can update a task